src/UI/ComposeEditor/Keybindings.hs

```diff
@@ -87,7 +87,8 @@ listOfAttachmentsKeybindings =
     , Keybinding (V.EvKey (V.KChar 'k') []) (listUp `chain` continue)
     , Keybinding (V.EvKey (V.KChar 'G') []) (listJumpToEnd `chain` continue)
     , Keybinding (V.EvKey (V.KChar '1') []) (listJumpToStart `chain` continue)
-    , Keybinding (V.EvKey (V.KChar 'y') []) (done `focus` continue @'Threads @'ListOfThreads)
+    , Keybinding (V.EvKey (V.KChar 'y') [])
+        (ifte done (switchView @'Threads @'ListOfThreads) noop *> continue)
```

Comment:
The first action is the boolean action? Then we have `then` and third is the `else`? See while `done` makes sense everywhere else, I guess here it might be better we use the actual `sendmail` action or whatever we've wired in the `CompletableResult`, just because here it would be more explicit of what we're actually doing here.  Not sure what your thoughts are.

Reply:
I think that's reasonable, but it's doesn't need to be done as part of this PR.

Reply:
Yeah fair enough...